Murcielago/Murcielago/UserInterface/Views/SignInView/SignInView.swift

```diff
@@ -12,6 +12,7 @@ struct SignInView: View {
     @State var usernameInput: String = ""
     @State var passwordInput: String = ""
     @State var showLoginUserInterface = false
+    @FocusState private var isFocused: Bool
```

Comment:
Why isn't this initialized to true/false?

Reply:
This property wrapper hooks in to SwiftUIs focus state and is automatically managed